simulation/World/world.py

```diff
@@ -0,0 +1,59 @@
+# import random
+# import numpy as np
+# from simulation.Vehicle.rule_based import RuleBased
+from simulation.Map.map import Map
+
+
+class World(object):
+
+    def __init__(self, rb_size, no_of_lanes, road_length, data):
+        self.rb_size = 0
```

Comment:
1 )The naming convention should be obvious otherwise add a comment. 
2) make them private as well.